api_tests/applications/views/test_application_list.py

```diff
@@ -1,45 +1,47 @@
+import pytest
 import copy
 import mock
 
-from nose.tools import *  # flake8: noqa
-
 from website.models import ApiOAuth2Application, User
 from website.util import api_v2_url
 from website.util import sanitize
-
 from tests.base import ApiTestCase
 from osf_tests.factories import ApiOAuth2ApplicationFactory, AuthUserFactory
 
 
 def _get_application_detail_route(app):
-    path = "applications/{}/".format(app.client_id)
+    path = 'applications/{}/'.format(app.client_id)
     return api_v2_url(path, base_route='/')
 
 
 def _get_application_list_url():
-    path = "applications/"
+    path = 'applications/'
     return api_v2_url(path, base_route='/')
 
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
 
-class TestApplicationList(ApiTestCase):
-    def setUp(self):
-        super(TestApplicationList, self).setUp()
-
-        self.user1 = AuthUserFactory()
-        self.user2 = AuthUserFactory()
+@pytest.mark.django_db
+@pytest.mark.usefixtures('app')
+class TestApplicationList:
 
-        self.user1_apps = [ApiOAuth2ApplicationFactory(owner=self.user1) for i in xrange(3)]
-        self.user2_apps = [ApiOAuth2ApplicationFactory(owner=self.user2) for i in xrange(2)]
+    @pytest.fixture()
+    def user_apps(self, user):
```

Comment:
Do we actually need to create multiple apps? I think it would make sense to just have a singular "user_app" fixture, then create more ApiOauth2Application objects in any test that requires more than one.